### PR TITLE
Windows Network/Make/Model/Serial/Total Memory metadata

### PR DESCRIPTION
**Pull request by gbrayut**
Network metadata for IP address is currently blocked by https://github.com/StackExchange/wmi/issues/5
Network metadata for using same iface tag is blocked by https://github.com/bosun-monitor/scollector/issues/119

### CODE REVIEW

File: metadata/metadata_windows.go [hunk 1]
```diff
@@ -32,19 +59,66 @@ type Win32_OperatingSystem struct {
...
+
+	for _, v := range dstAdapters {
+		tag := opentsdb.TagSet{"iface": v.InterfaceName} //Should be v.Name. See https://github.com/bosun-monitor/scollector/issues/119
+		AddMeta("", tag, "description", v.InterfaceDescription, true)
+		AddMeta("", tag, "name", v.Name, true)
+		AddMeta("", tag, "speed", v.Speed, true)
+
+		nicConfig := mNicConfigs[v.InterfaceGuid]
+		if nicConfig != nil {
+			AddMeta("", tag, "mac", strings.Replace(nicConfig.MACAddress, ":", "", -1), true)
```
Comment (madelynnblue) [inline metadata/metadata_windows.go:102]: Why do we need to remove : from MAC addresses?

File: metadata/metadata_windows.go [hunk 2]
```diff
@@ -32,19 +59,66 @@ type Win32_OperatingSystem struct {
...
+			for _, ip := range *nicConfig.IPAddress {
+				fmt.Println("%v", ip)
```
Comment (madelynnblue) [inline metadata/metadata_windows.go]: Remove this debug. And for the future, just `fmt.Println(ip)` is correct. fmt.Printf uses the format string.

File: metadata/metadata_windows.go [hunk 3]
```diff
@@ -32,19 +59,66 @@ type Win32_OperatingSystem struct {
...
+	InterfaceDescription string //Intel(R) Gigabit ET Quad Port Server Adapter #2
+	InterfaceName        string //Ethernet_10
+	InterfaceGuid        string //unique id
+}
+
+type Win32_NetworkAdapterConfiguration struct {
+	IPAddress  *[]string //Both IPv4 and IPv6
+	MACAddress string    //00:1B:21:93:00:00
+	SettingID  string    //Matches InterfaceGuid
+	Caption    string
```
Comment (madelynnblue) [inline metadata/metadata_windows.go]: This is unused, can remove.

File: metadata/metadata_windows.go (hunk 1 above)
Comment (gbrayut) [inline metadata/metadata_windows.go:102]: All instances of MAC in tags has : removed. Also https://github.com/bosun-monitor/bosun/issues/386 uses the same format. I'm ok with leaving : in but assumed we would want the metadata to match other instances where we use it.

Comment (madelynnblue) [inline metadata/metadata_windows.go:102]: Ok, that's fine.

**Pull request merged**